components/coding/ExpandableContainer.tsx

```diff
@@ -35,7 +39,9 @@ const ExpandableContainer: React.FC<ExpandableContainerProps> = ({
                 <MinusCircleIcon className="w-6 h-6 cursor-pointer hover:text-yellow-600" />
               )}
             </button>
-            <h6 className="">{title}</h6>
+            <h6 className="text-3xl font-bold pb-2 md:text-4xl mt-3">
```

Comment:
design nit: text-3xl looks a little awkward with the profile header font, maybe something less than this?

<img width="799" alt="Screen Shot 2023-01-09 at 2 22 24 PM" src="https://user-images.githubusercontent.com/98791180/211390492-03dae756-3083-46ca-b0a9-4698fe3ee5ad.png">

Reply:
reduced the title size by one...looks a lot better.